examples/manipulator/children.py

```diff
@@ -169,14 +183,21 @@ def _rotate_rect(self, dt):
 
     def construct_manipulator(self):
         material = Material(
-            color=(0.0, 0.0, 1.0), diffuse=(1.0, 1.0, 0.0), specular=(0.35, 0.35, 0.35)
+            color=(0.0, 0.0, 1.0),
+            diffuse=(1.0, 1.0, 0.0),
+            specular=(0.35, 0.35, 0.35)
         )
 
-        # axis a
-        # one jaw
-        axis_a_dimensions = JOINT_SIZE_A[0], JOINT_SIZE_A[2], JOINT_SIZE_A[1]
+        # axis a one jaw
+        axis_a_dimensions = (
+            JOINT_SIZE_A[0],
+            JOINT_SIZE_A[2],
+            JOINT_SIZE_A[1]
+        )
         axis_a_geometry = create_joint_rectangle(
-            axis_a_dimensions[0], axis_a_dimensions[1], axis_a_dimensions[2]
+            axis_a_dimensions[0],
+            axis_a_dimensions[1],
+            axis_a_dimensions[2]
```

Comment:
Untested suggestion:
```suggestion
            *axis_a_dimensions
```
Same applies for other similar calls below

Reply:
done